pkg/apis/servicecatalog/types.go

```diff
@@ -266,10 +266,16 @@ type InstanceSpec struct {
 	// provisioned from.
 	PlanName string
 
-	// Parameters is a YAML representation of the properties to be
+	// Parameters is set of properties to be
 	// passed to the underlying broker.
 	Parameters *runtime.RawExtension
 
+	// Optional: AlphaSecretParameters is reference to the secret object containing
+	// sensitive information to pass to the underlying broker. This may be
+	// empty if no secret object is specified.
+	// +optional
+	AlphaSecretParameters *v1.LocalObjectReference
```

Comment:
> just FYI but I did have a chat with brian grant a while ago and he was ok with us using secrets to store this info.

Glad to hear that. I personally also don't understand why there is so much fuss about this. IMO using Secrets for different use cases lines up with Secrets roadmap (plus what has been discussed at SIG Auth meetings) perfectly.
But I agree with Paul that we need an "official green light" to avoid any confusion in future.